### PR TITLE
Create rameez.json

### PR DESCRIPTION
**Pull request by rameezrami**

### CODE REVIEW

File: domains/rameez.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal website of Rameez",
+  "repo": "https://github.com/rameezrami/rameezrami.github.io",
+  "owner": {
+    "username": "rameezrami",
+    "email": "rameespu@gmail.com"
+  },
+  "record": {
+    "URL": "https://rameez.is-a.dev"
```
Comment (phenax) [inline domains/rameez.json:9]: As I mentioned in the old PR, `rameez.is-a.dev` cant be redirected to `https://rameez.is-a.dev`.

For URL redirection
```suggestion
    "URL": "https://rameezrami.github.io"
```

For subdomain
```suggestion
    "CNAME": "rameezrami.github.io"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Comment by rameezrami** [comment]
updated 

"record": {
	"URL": "https://rameezrami.github.io",
	"CNAME": "rameezrami.github.io"
  }

**Review by rameezrami — COMMENTED** [review]
all good now?

File: domains/rameez.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal website of Rameez",
+  "repo": "https://github.com/rameezrami/rameezrami.github.io",
+  "owner": {
+    "username": "rameezrami",
+    "email": "rameespu@gmail.com"
+  },
+  "record": {
+    "URL": "https://rameezrami.github.io",
+	  "CNAME": "rameezrami.github.io"
```
Comment (phenax) [inline domains/rameez.json:10]:
```suggestion
    "CNAME": "rameezrami.github.io"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**